### DISCUSSION TITLE
add virtual_ai_assistant_with RAG_demo

### DISCUSSION THREAD

**Discussion by nrichaud** [openvinotoolkit/openvino_build_deploy#93] [Ideas]
I like very much the "virtual_ai_assistant_demo". It would be great if with the personality.yaml file, we can define a list of document to "ingest".
Alternatively a kind of 2 steps approach where we create/update a vectoreDB and then include this vector DB as part of the personality.yaml.

**Comment by raymondlo84Fork** [comment]
Thank you for the feedback. Also always feel free to make a pull request and update it =).

FYI @adrianboguszewski

**Reply by adrianboguszewski** [reply]
Yep, I have that in my list to add sample documents as a parameter :)

**Comment by raymondlo84Fork** [comment]
@openvino-dev-samples  FYI Ethan too
